lib/dynamoid/criteria/chain.rb

```diff
@@ -156,9 +156,10 @@ def records_via_scan
       end
 
       def range_hash(key)
-        val = query[key]
+        attr, operation = key.to_s.split('.')
```

Comment:
@andrykonchin Sorry, I misread my documentation tool.  It is core Ruby, not Rails.  It is defined in the core Ruby `Module` object.

http://ruby-doc.org/core-2.4.1/Module.html#method-i-attr